src/coreclr/jit/gentree.h

```diff
@@ -492,6 +493,7 @@ enum GenTreeFlags : unsigned int
     GTF_MEMORYBARRIER_LOAD      = 0x40000000, // GT_MEMORYBARRIER -- Load barrier
 
     GTF_FLD_TLS                 = 0x80000000, // GT_FIELD_ADDR -- field address is a Windows x86 TLS reference
+    GTF_FLD_NEVER_NEGATIVE      = 0x80000000, // GT_FIELD -- this field can be assumed to never return a negative value
```

Comment:
We could track this one differently since `GenTreeField` has a `bool` field and therefore `3 padding bytes`.

Doing the same for `VAR` would be more difficult since it doesn't have "free space" like that. But things could likely be moved around slightly if we thought it was worthwhile.

Reply:
PR was indeed updated to track this differently.